assertion_utils/index.ts

```diff
@@ -0,0 +1,39 @@
+
+export function forDataset(foo: string) {
+  return new DatasetAssertion(foo);
+}
+
+export class DatasetAssertion {
+  private readonly dataset: string;
+  private groupCols: string[] = [];
+  constructor(dataset: string) {
+    this.dataset = dataset;
+  }
+
+  public groupedBy(cols: string[]) {
+    this.groupCols = cols;
+    return this;
+  }
+
+  public getUniqueRowQuery(): string {
+    return `
+    WITH base AS (
```

Comment:
is this bigquery-formatted? might be nice to follow that.

Reply:
Opting to ignore this one :)